web-animations-1/Overview.bs

```diff
@@ -1324,6 +1307,12 @@ follows:
     perform the steps according to the first matching condition from below:
 
     <div class="switch">
+    :   If |animation|'s <a>start time</a> is <a>unresolved</a> and
+        <em>either</em>:
+        *   the |animation| has no associated <a>timeline</a>, or
+        *   the associated <a>timeline</a> is
+            <a lt="inactive timeline">inactive</a>
```

Comment:
As with my first comment, I'm afraid I don't follow why we have these particular conditions.

Reply:
The explanation is the same as in the first comment.

Reply:
Please see the first comment.